### ISSUE TITLE
MSB6006 Exit Code 1 (Microsoft.VisualBasic.Core.targets(59,5))

### ISSUE THREAD

**Issue by AdamSpeight2008** [dotnet/roslyn#12690]
There seems to be an issue with `Microsoft.VisualBasic.Core.targets(59,5)`

http://dotnet-ci.cloudapp.net/job/dotnet_roslyn/job/master/job/windows_debug_unit32_prtest/1525/consoleFull#-178546567179494335-f7bd-47d0-8771-8661e00c2db2

http://dotnet-ci.cloudapp.net/job/dotnet_roslyn/job/master/job/windows_debug_unit64_prtest/1532/console

**Comment by AdamSpeight2008** [comment]
`(59,5)` is the opening angle brace of `<Vbc`
```xml
    <!-- Condition is to filter out the _CoreCompileResourceInputs so that it doesn't pass in culture resources to the compiler -->
    <Vbc  Condition=" '%(_CoreCompileResourceInputs.WithCulture)' != 'true' "
          AdditionalLibPaths="$(AdditionalLibPaths)"
          AddModules="@(AddModules)"
          AdditionalFiles="@(AdditionalFiles)"
          Analyzers="@(Analyzer)"
          BaseAddress="$(BaseAddress)"
          ChecksumAlgorithm="$(ChecksumAlgorithm)"
          CodeAnalysisRuleSet="$(ResolvedCodeAnalysisRuleSet)"
          CodePage="$(CodePage)"
          DebugType="$(DebugType)"
          DefineConstants="$(FinalDefineConstants)"
          DelaySign="$(DelaySign)"
          DisabledWarnings="$(NoWarn)"
          DocumentationFile="@(DocFileItem)"
          EmitDebugInformation="$(DebugSymbols)"
          EnvironmentVariables="$(VbcEnvironment)"
          ErrorLog="$(ErrorLog)"
          ErrorReport="$(ErrorReport)"
          Features="$(Features)"
          FileAlignment="$(FileAlignment)"
          GenerateDocumentation="$(GenerateDocumentation)"
          HighEntropyVA="$(HighEntropyVA)"
          Imports="@(Import)"
          KeyContainer="$(KeyContainerName)"
          KeyFile="$(KeyOriginatorFile)"
          LangVersion="$(LangVersion)"
          LinkResources="@(LinkResource)"
          MainEntryPoint="$(StartupObject)"
          ModuleAssemblyName="$(ModuleAssemblyName)"
          NoConfig="true"
          NoStandardLib="$(NoCompilerStandardLib)"
          NoVBRuntimeReference="$(NoVBRuntimeReference)"
          NoWarnings="$(_NoWarnings)"
          NoWin32Manifest="$(NoWin32Manifest)"
          Optimize="$(Optimize)"
          Deterministic="$(Deterministic)"
          PublicSign="$(PublicSign)"
          OptionCompare="$(OptionCompare)"
          OptionExplicit="$(OptionExplicit)"
          OptionInfer="$(OptionInfer)"
          OptionStrict="$(OptionStrict)"
          OptionStrictType="$(OptionStrictType)"
          OutputAssembly="@(IntermediateAssembly)"
          PdbFile="$(PdbFile)"
          Platform="$(PlatformTarget)"
          Prefer32Bit="$(Prefer32Bit)"
          PreferredUILang="$(PreferredUILang)"
          ProvideCommandLineArgs="$(ProvideCommandLineArgs)"
          References="@(ReferencePath)"
          RemoveIntegerChecks="$(RemoveIntegerChecks)"
          ReportAnalyzer="$(ReportAnalyzer)"
          Resources="@(_CoreCompileResourceInputs);@(CompiledLicenseFile)"
          ResponseFiles="$(CompilerResponseFile)"
          RootNamespace="$(RootNamespace)"
          RuntimeMetadataVersion="$(RuntimeMetadataVersion)"
          SdkPath="$(FrameworkPathOverride)"
          SkipCompilerExecution="$(SkipCompilerExecution)"
          Sources="@(Compile)"
          SubsystemVersion="$(SubsystemVersion)"
          TargetCompactFramework="$(TargetCompactFramework)"
          TargetType="$(OutputType)"
          ToolExe="$(VbcToolExe)"
          ToolPath="$(VbcToolPath)"
          TreatWarningsAsErrors="$(TreatWarningsAsErrors)"
          UseHostCompilerIfAvailable="$(UseHostCompilerIfAvailable)"
          UseSharedCompilation="$(UseSharedCompilation)"
          Utf8Output="$(Utf8Output)"
          VBRuntimePath="$(VBRuntimePath)"
          Verbosity="$(VbcVerbosity)"
          VsSessionGuid="$(VsSessionGuid)"
          WarningsAsErrors="$(WarningsAsErrors)"
          WarningsNotAsErrors="$(WarningsNotAsErrors)"
          Win32Icon="$(ApplicationIcon)"
          Win32Manifest="$(Win32Manifest)"
          Win32Resource="$(Win32Resource)"
          VBRuntime="$(VBRuntime)"
          PathMap="$(PathMap)"
          >
      <Output TaskParameter="CommandLineArgs" ItemName="VbcCommandLineArgs" />
    </Vbc>
```

**Comment by AdamSpeight2008** [comment]
`Roslyn.ExpressionEvaluator.VisualBasic.ExpressionCompiler.UnitTests.dll.xml`
`Roslyn.ExpressionEvaluator.CSharp.ExpressionCompiler.UnitTest.dll.xml`
```xml
<?xml version="1.0" encoding="utf-8"?>
<assemblies>
  <assembly />
</assemblies>
```
`Roslyn.ExpressionEvaluator.VisualBasic.ExpressionCompiler.UnitTests.dll.out.log`

```
xUnit.net Console Runner (32-bit .NET 4.0.30319.42000)
System.IO.FileLoadException: Could not load file or assembly 'Microsoft.VisualStudio.Debugger.Engine, Version=1.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. Strong name validation failed. (Exception from HRESULT: 0x8013141A)
System.Security.SecurityException: Strong name validation failed. (Exception from HRESULT: 0x8013141A)
```
`Roslyn.ExpressionEvaluator.CSharp.ExpressionCompiler.UnitTest.dll.out.log`
```
xUnit.net Console Runner (32-bit .NET 4.0.30319.42000)
System.IO.FileLoadException: Could not load file or assembly 'Microsoft.VisualStudio.Debugger.Engine, Version=1.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. Strong name validation failed. (Exception from HRESULT: 0x8013141A)
System.Security.SecurityException: Strong name validation failed. (Exception from HRESULT: 0x8013141A)
```

**Comment by lorcanmooney** [comment]
Is this a dupe of #12539 (which I'm still seeing)?

**Comment by AdamSpeight2008** [comment]
@lorcanmooney If it is we maybe able to narrow down the range of when it a occurred, via the merge dates.

**Comment by AdamSpeight2008** [comment]
Seems to be fixed via (PR  #12772 )

**Comment by Rmtdi** [comment]
hi, are you fixed it?
I have the same problem about exit code 1 and Microsoft.core.target. Can you help me?
